packages/ubuntu_desktop_installer/lib/installer.dart

```diff
@@ -37,6 +41,11 @@ class UbuntuDesktopInstallerApp extends StatelessWidget {
       home: _UbuntuDesktopInstallerWizard.create(context),
     );
   }
+
+  static final _appName = p.basename(io.Platform.resolvedExecutable);
+
+  /// Static reference to an application-level common [Logger].
+  static final log = Logger(_appName);
```

Comment:
what if we had this as a global in `logger.dart`? then we could use `log.foo()` everywhere and it would default to the application-wide logger, but subiquity_client and others could still use a local logger instance.

Reply:
Good suggestion. I named the common application logger `appLogger`, because I feel `log` is a bit too generic, but I don't have a strong opinion on this, so let me know if you'd rather rename it.

Reply:
I'd vote for `log` because then you wouldn't need to stop to think and look around, but you'd automatically get the nearest logger in the scope. :slightly_smiling_face: 

I'm not against a general app-wide logger as a fallback, but I had somehow envisioned that we'd use smaller scoped loggers. The `<app-name>` context is redundant because the log-file already tells that. Smaller scopes would make it clear where the output comes without needing to specify that in the output itself.

Reply:
> I had somehow envisioned that we'd use smaller scoped loggers.

For example, everything in `pages/keyboard_layout/*.dart` would log to `Logger('keyboard_layout')`.

Reply:
Ah, that makes sense, I'll update the PR to implement it.

Reply:
Regarding the automatic scoping discussion on Friday. Perhaps we don't actually need anything fancy like that, because if we manage to keep views dumb and declarative, all(?) logging would be in models and services.

Reply:
Agreed. I went back on my initial proposal, and used per-file `Logger` instances to log relevant information, only in models and services.